HHS/Head-Start-TTADP
# [TTAHUB-2709] Fix bug with goal status, display goals missing information

## Description of change
- Add change demoed in standup
- do not explicitly set status in goal services when saving goals for report
- Update report.updatedAt when a response is changed, so that local storage is invalidated

## How to test

This process allows you to submit 40249

- Load most recent prod data from keybase
- Impersonate user 112 (report creator)
- Open report 40249
- Copy completed goal and objective data to txt file
- Delete all goals on report
- Add FEI goal and objective data back to report
- Save
- Go to review
- Add FEI data to each goal there
- Refresh
- Confirm that the closed goal is actually removed from report
- If not, remove and save
- You should be able to submit report at this point

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2709


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
